src/scripts.js

```diff
@@ -39,37 +36,38 @@ hydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydratio
   .then(data => data.hydrationData)
   .catch(error => console.log('hydrationData error'))
 
-// We need to somehow make sure that we have access to all
-// of the methods in every class. So when we call hydration.Method(),
-// it knows what hydration is and that it's supposed to reach
-// into that class and execute a method from there.
-
+// PROMISE
 Promise.all([userData, sleepData, activityData, hydrationData])
   .then(data => {
     userData = data[0];
     sleepData = data[1];
     activityData = data[2];
     hydrationData = data[3];
+    console.log('data', userData)
   })
   .then(() => {
-    userRepository = new UserRepository();
+    userRepository = new UserRepository(userData, sleepData, activityData, hydrationData);
     instantiateAllUsers();
     instantiateAllUsersActivity();
     instantiateAllUsersHydration();
     instantiateAllUsersSleep();
   })
   .then(() => {
     let user = userRepository.users[Math.floor(Math.random() * userRepository.users.length)]
-    let todayDate = "2019/09/22";
+    let todayDate = "2020/01/22";
```

Comment:
updating current date from dataset.  Will go back to make it dynamic once we can start posting new data into the database.